infra/recipes/terraform/azure/modules/mysql/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_subnet" "mysql_subnet" {
+  name = var.subnet_name
+  resource_group_name = var.resource_group_name
+  virtual_network_name = var.virtual_network_name
+  address_prefixes = ["10.0.0.0/24"]
+
+  delegation {
+    name = "delegation"
+
+    service_delegation {
+      name = "Microsoft.DBforMySQL/flexibleServers"
+      actions = ["Microsoft.Network/networkinterfaces/*"]
+    }
+  }
+}
+
+resource "azurerm_mysql_flexible_server" "mysql_server" {
+  name = var.server_name
+  resource_group_name = var.resource_group_name
+  location = var.resource_group_location
+  administrator_login = var.db_admin_login
+  administrator_password = var.db_admin_password
+  delegated_subnet_id = azurerm_subnet.mysql_subnet.id
+  sku_name = "GP_Standard_D2ds_v4"
```

Comment:
Seeing that default vm_size used is `Standard_D2s_v3`. Is sku `GP_Standard_D2ds_v4` is specifically for mysql configuration or can be exacted in variable if reusable?

Reply:
I am not fully sure. I will extract it as a variable.